### ISSUE TITLE
Calling SNPs on non-fastq files

### ISSUE THREAD

**Issue by nknox** [apetkau/core-phylogenomics#16]
When users want to included genomes for which they do not have access to fastq files - the current pipeline runs nucmer (pair-wise alignment) and calls SNPs using mummer. Ideally, reads should be simulated and FreeBayes/mpileup used to call variants.

**Comment by apetkau** [comment]
I agree that this would be a good way to go.  Another good idea for a project.

For some of these components, I'd like to see them implemented as separate modules in a separate git repository.  That we it makes integration into Galaxy a bit easier (we can include the Galaxy tool definition files) and gives a bit more separation.
